src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -23,39 +24,53 @@
             + "Example: " + COMMAND_WORD + " 1";
 
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person(s):\n%1$s";
 
-    private final Index targetIndex;
+    private ArrayList<Index> targetIndexArraylist;
+
+    public DeleteCommand(ArrayList<Index> targetIndexArraylist) {
+        this.targetIndexArraylist = targetIndexArraylist;
+    }
 
     public DeleteCommand(Index targetIndex) {
```

Comment:
I presume you left this constructor here for usage in Tests.
You can actually omit this constructor and change the respective codes in tests e.g.:

from:
`DeleteCommand deleteCommand = prepareCommand(INDEX_FIRST_PERSON);`
to:
`DeleteCommand deleteCommand = prepareCommand(Arrays.asList(INDEX_FIRST_PERSON));`